docs/SchemaMigrationTool.md

```diff
@@ -4,7 +4,20 @@ The SQL Server schema migration tool is the command line utility to perform SQL
 Note - The tool can't downgrade a schema version.
 
 - #### Prerequisites
-    The tool would only run when the current schema is in conformance to the [BaseSchema](BaseSchema.md). So either the BaseSchema should already be present or if not, then Schema admin would need to run BaseSchema script manually.
+    The FHIR Server for Azure has the 
+    - minimum supported schema version >= 4 and 
+    - the current schema version is null or >= 3.
+
+    Note: If the FHIR Server for Azure is running on current schema version < 3(i.e. 1 or 2), manual intervention is required to upgrade the current schema version to 3.
+
+    #### Manual steps to upgrade current schema version(1 or 2) to 3 using any SQL Editor
+    - If the current schema version is 1, then
+        1. Execute the content of 2.diff.sql.
+        2. After step(a) is success, execute the query 'INSERT INTO dbo.SchemaVersion VALUES (2, 'completed')'
```

Comment:
Is step(a) step 1? Perhaps reword `After step 1 is complete`

Reply:
Its little weird that in the preview it is getting converted into letters.
![image](https://user-images.githubusercontent.com/57157506/94072189-11048d00-fdaa-11ea-8079-deb3ad944078.png)

Reply:
After finding little bit more on this, numbers are getting converted due to nested structure. So to be consistent with .md and its preview, have reworded After step(a) => After previous step.